marklogic-data-hub-spark-connector/src/test/java/com/marklogic/hub/spark/sql/sources/v2/Options.java

```diff
@@ -21,8 +21,7 @@
     private String permissions;
     private String sourceName;
     private String sourceType;
-    private JsonNode ingestWorkUnit;
-    private JsonNode ingestEndpointState;
+    private JsonNode ingestEndpointConstants;
```

Comment:
Don't we still need ingestEndpointState? Or is that not relevant at all to input-only endpoints?

Reply:
Done.